Conceptpower+Spring/src/integration-test/java/edu/asu/conceptpower/IntegrationTest.java

```diff
@@ -54,26 +45,10 @@ public void setup() throws Exception {
 
         this.mockMvc = MockMvcBuilders.webAppContextSetup(this.wac).build();
         if (!isSetupDone) {
-            setSecurityContext();
             logger.debug("Started with lucene indexing for integration tests.");
             this.mockMvc.perform(MockMvcRequestBuilders.post("/auth/indexConcepts").principal(principal));
             isSetupDone = true;
             logger.debug("Completed with lucene indexing for integration tests.");
         }
     }
-
-    private void setSecurityContext() {
-        logger.debug("Setting up security context");
-        Collection<ConceptpowerGrantedAuthority> authorities = new ArrayList<>();
-        ConceptpowerGrantedAuthority grantedAuthority = new ConceptpowerGrantedAuthority("admin");
-        grantedAuthority.setAuthority(ConceptpowerGrantedAuthority.ROLE_ADMIN);
-        authorities.add(grantedAuthority);
-
-        AbstractAuthenticationToken auth = new AnonymousAuthenticationToken("admin", principal, authorities);
-        SecurityContext context = new SecurityContextImpl();
-        context.setAuthentication(auth);
-        SecurityContextHolder.setContext(context);
-        logger.debug("Completed setting up security context");
-    }
```

Comment:
We are not providing the security context because I am not including the spring-security.xml while building the application for integration test. I tested by just running the application on tomcat, the add/concepts restful call is secure. Let me know if I need to include the spring-security.xml in integration test and include the security contexts as well while testing?